theia/utils/panoptes_oauth2.py

```diff
@@ -22,13 +22,21 @@ class PanoptesOAuth2(BaseOAuth2):
     ]
 
     def get_user_details(self, response):
-        with Panoptes() as p:
-            p.bearer_token = response['access_token']
-            p.logged_in = True
-            p.refresh_token = response['refresh_token']
-            p.bearer_expires = (datetime.now() + timedelta(seconds=response['expires_in']))
+        authenticated_panoptes = Panoptes(
```

Comment:
Noting that this request will return the user details of zooniverse oauth application owner, not the actual user that is initiating the request (unless they own the oauth application). 

Unless i'm missing how the `Panoptes` social module works, normally client creds flow returns the token for the owning user.